frontend/.eslintrc.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
```

Comment:
might want to add your linter to gitignore.

Reply:
Normally linter configuration is part of the project as a team usually shares the same linter rules.